packages/components/src/widgets/DistWidget/SummaryTable.tsx

```diff
@@ -58,7 +58,7 @@ const SummaryTableRow: FC<SummaryTableRowProps> = ({
     if (valueType == "date") {
       // When dealing with dates, the standard deviation is a duration, not a date, so we need to format it differently
       if (isRange) {
-        return duration.toString(number);
+        return SDuration.fromMs(number).toString();
```

Comment:
This is arguably a roundabout way of doing this, there could be a better approach, or we might choose to duplicate logic.

Reply:
I still think that you could go through `SqDurationValue.createFromMs(number).toString()` and it would be better than exposing `SDuration` and `SDate`, but it's less of a concern for me now that names are more unique.